cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -640,6 +640,13 @@ export class Application {
         Assessment.validateReviewFields(this.name, "Application");
     }
 
+    validateInheritedReviewFields(archetypes): void {
+        Application.open();
+        for (let i in archetypes) {
+            Assessment.validateReviewFields(this.name, "Archetype", archetypes[i].name);
+        }
```

Comment:
Your approach is perfectly valid, if you want to avoid having to type `[i]` you can use a `for of` to make use of the object instead of the index

```
for (let archetype of archetypes) {
    Assessment.validateReviewFields(this.name, "Archetype", archetype.name);
}
```